### ISSUE TITLE
subete is extremely noisy

### ISSUE THREAD

**Issue by rzuckerm** [TheRenegadeCoder/subete#63]
For the sample-programs-website, about 100K+ lines are output. It would be nice if this were reduced to only what is necessary to show the progress is being made. I think one way to do this would be to remove logging from the getter methods.

**Comment by jrg94** [comment]
Makes sense to me! Another option would be to use the logging levels appropriately, so the noisy bits can be hidden from the output but switched on if needed.

**Comment by rzuckerm** [comment]
> Makes sense to me! Another option would be to use the logging levels appropriately, so the noisy bits can be hidden from the output but switched on if needed.

Yeah, that's probably easier. Just have the automation set it to `WARNING` instead of `INFO` and have the automation log appropriate messages.
